monitoring/update-bucket-capacity-info-cronjob/alerts.yaml

```diff
@@ -0,0 +1,99 @@
+# Variables which should be replaced. Similar to grafana dashboards' __inputs section
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: update_bucket_capacity_info_cronjob
+    type: constant
+    value: artesca-data-ops-update-bucket-capacity-info
+  - name: update_bucket_capacity_info_job_duration_threshold
+    type: config
+    value: 240
+  - name: update_bucket_capacity_info_success_job_existence_duration_threshold
+    type: config
+    value: 600
+
+groups:
+  - name: update-bucket-capacity-info-cronjob/alerts.rules
+    rules:
+      - alert: UpdateBucketCapacityJobTakingTooLong
+        expr: |
+          (
+            time() - 
+            label_replace(
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${update_bucket_capacity_info_cronjob}"}
+              )
+              BY (job_name, owner_name, namespace)
+              == ON(owner_name) GROUP_LEFT()
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${update_bucket_capacity_info_cronjob}"}
+              )
+              BY (owner_name),
+              "cronjob", "$1", "owner_name", "(.*)")
+          ) > ${update_bucket_capacity_info_job_duration_threshold}
+        labels:
+          severity: critical
+        annotations:
+          description: "Job {{$labels.job_name}} is taking more than 4m to complete"
```

Comment:
nit: I would add a description on the impacts. (un-accuracy of metrics and Veeam integration at risk)